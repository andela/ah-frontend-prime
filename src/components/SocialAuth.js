import React from "react";
import FacebookLogin from "react-facebook-login";
import GoogleLogin from "react-google-login";
import config from "../../src/config.json";

export const SocialAuth = props => (
  <div>
    <div className="face-book">
      <FacebookLogin
        appId={config.appId}
        fields="name,email,picture"
        callback={props.facebooksuccess}
        icon={
          <img
            src="https://img.icons8.com/color/96/000000/facebook.png"
            width="=40px"
            height="40px"
            alt="facebook"
          />
        }
        textButton="SIGN IN WITH FACEBOOK"
      />
    </div>
    <br />
    <div className="google">
      <GoogleLogin
        clientId={config.clientId}
        buttonText="SIGN IN WITH GOOGLE"
        onSuccess={props.googlesuccess}
        onFailure={props.googlefailure}
      />
    </div>
  </div>
);

export default SocialAuth;
