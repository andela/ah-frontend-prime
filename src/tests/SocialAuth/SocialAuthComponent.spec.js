import React from "react";
import { shallow } from "enzyme";
import SocialAuth from "../../components/SocialAuth";

describe("6", () => {
  it("should render without crashing", () => {
    const wrapper = shallow(<SocialAuth />);
    expect(wrapper).toMatchSnapshot();
  });
});
