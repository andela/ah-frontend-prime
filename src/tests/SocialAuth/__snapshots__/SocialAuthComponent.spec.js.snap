// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`6 should render without crashing 1`] = `ShallowWrapper {}`;
